supervisor/shared/bluetooth/file_transfer.c

```diff
@@ -131,12 +133,23 @@ void supervisor_start_bluetooth_file_transfer(void) {
 #define ANY_COMMAND 0x00
 #define THIS_COMMAND 0x01
 
+uint64_t truncate_time(uint64_t input_time, DWORD *fattime) {
```

Comment:
Could you add a comment explaining the purpose of this? It's implied in the README (I had to look it up), but I think it would be good to mention the 2-second FAT granularity here, since it's pretty obscure what's going on otherwise.